### ISSUE TITLE
Any plan to release training details such as hyper-parameters(epoch, batch, learning rate) and loss curve

### ISSUE THREAD

**Issue by purezhanghan** [antgroup/Agentar-Scale-SQL#16]
Thanks for open-sourcing the project and model.

**Comment by Peng-fei-Wang** [comment]
Hi @purezhanghan , thanks for your interest in our work!

We plan to include more training details, such as the hyper-parameters you mentioned, in a future revision of our paper. Please keep an eye on the paper for updates.

**Comment by Peng-fei-Wang** [comment]
@purezhanghan 
Detailed hyperparameters and model configurations are provided in the table below:

Category | Hyperparameter | Generator | Selector
-- | -- | -- | --
Data & Model | Base Model | OmniSQL-32B | Qwen2.5-Coder-32B
  | Max Prompt Length | 9216 | 9216
  | Max Response Length | 2048 | 2048
GRPO Algorithm | Reward Function | syntax, execution | selection
  | Mini-Batch Size | 64 | 128
  | Micro-Batch Size / GPU | 1 | 2
  | KL Divergence Coefficient | 0.001 | 0.001
  | Learning Rate | 1e-6 | 1e-6
  | LR Warmup Steps Ratio | 0.1 | 0.1
Rollout | Rollout Engine | SGLang | SGLang
  | GPU Memory Utilization | 0.5 | 0.8
  | N | 16 | 16
  | Log-Prob Micro-Batch Size / GPU | 1 | 2
  | Ref Model Log-Prob Micro-Batch / GPU | 1 | 2
  | Tensor Parallel Size (TP) | 8 | 8
Training | Total Epochs | 3 | 5
  | Total GPUs | 32 | 32
  | GPUs per Node | 8 | 8
  | Total Nodes | 4 | 4
  | GPU Type | A100 | A100
